open-metadata-implementation/repository-services/repository-services-implementation/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
The new code uses the `ImmutableList` class from Guava.  It is not strictly necessary, I can remove it.  I assumed that since it was already being used it had been approved.

Reply:
Guava gets used transitively by some test tools including testng. It's also used by janusgraph and cassandra. We have to explicitly specify the version to resolve version conflicts. Currently it is also used explicitly in open lineage services